client/dataHelper.js

```diff
@@ -69,70 +69,79 @@ export async function sendUpdatedPassword (token, oldPass, newPass) {
 
 /**
  * Retrieves a list of valid phrases from the database
- * 
+ *
  * @param {String} token The session token from the cookie system
  * @returns {Array} A list of phrases as an array
  */
 export async function getAllPhrases(token) {
     const getPhrases = await fetch("/api/getPhrases", {
         method: "POST",
         headers: {
-            "Content-Type": "application/json"
+            "Content-Type": "application/json",
         },
-        body: JSON.stringify({"token": token})
-    })
-    if(getPhrases.status === 400 || getPhrases.status === 401 || getPhrases.status === 500) {
-        window.alert((await getPhrases.text()).toString())
-    } else if(getPhrases.status === 200) {
-        return JSON.parse(await getPhrases.text())
+        body: JSON.stringify({ token: token }),
+    });
+    if (
+        getPhrases.status === 400 ||
+        getPhrases.status === 401 ||
+        getPhrases.status === 500
+    ) {
+        window.alert((await getPhrases.text()).toString());
+    } else if (getPhrases.status === 200) {
+        return JSON.parse(await getPhrases.text());
```

Comment:
Change this to respond only to code 200 and show a window alert for all other response codes.